docs/guide-tequila-api-key.md

```diff
@@ -1,6 +1,20 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
+
+To get your **API key** for the GraphQL got into the My applications page.
+Create the new app, by clicking on the `add application` button.
+
+Then you should choose your partnership the **<span>Kiwi</span>.com Affiliate Program** or **Book with <span>Kiwi</span>.com**. More information about partnership types, you will find there.
+
+After choosing your partnership, you need to select the product type, our GraphQL proxy support only the _Search_ and _Search & Book_ features, but the _Multicity_ and _Nomad_ are on our roadmap and will come later.
+
+When you have been created application you will be redirected to the application detail page where you can grab your key from the Detail section.
```

Comment:
Once you successfully created the application you will be redirected to the application detail page where you can grab your **API key** from the Detail section.

Reply:
done